includes/platform-checkout/class-platform-checkout-utilities.php

```diff
@@ -122,10 +123,16 @@ public function get_persisted_available_countries() {
 	 * @return array
 	 */
 	public function get_woopay_available_countries() {
-		$last_check = get_option( self::AVAILABLE_COUNTRIES_LAST_CHECK_KEY );
+		$last_check_option = get_option( self::AVAILABLE_COUNTRIES_LAST_CHECK_KEY );
+		$timezone          = new \DateTimeZone( wp_timezone_string() );
+		$current_date      = new \DateTime( 'now', $timezone );
 
-		if ( $last_check && gmdate( 'Y-m-d' ) === $last_check ) {
-			return $this->get_persisted_available_countries();
+		if ( isset( $last_check_option ) ) {
```

Comment:
If we check `isset` here, the first time `$last_check_option` === `false` but it's set so `isset` will return truthy.

Reply:
Nice catch, fixed on https://github.com/Automattic/woocommerce-payments/pull/6236/commits/c1735e745a648362f0305819184d0af5b17fb3cc.